overlay.yml

```diff
@@ -21,5 +18,10 @@ components:
     override-version: "1.13.1"
     branch: docker-1.13.1-rhel
 
+  # NOTE: master=1.11.0 which requires a different spec
+  - src: github:kubernetes-incubator/cri-o
+    override-version: "1.10.1"
```

Comment:
What happens without this? I see 1.10.1 tagged on the `release-1.10` branch. I thought rdgo would pick up on that.

Reply:
The result I got was 1.9 rpms with 1.10.1 content.